packages/keystone/databaseAdapters/BalancingReplicaKnexAdapter/adapter.js

```
@@ -13,179 +13,260 @@
+const DEFAULT_POOL_MAX = conf['DATABASE_POOL_MAX']
+    ? parseInt(conf['DATABASE_POOL_MAX'], 10)
+    : 3;
+
+const DEFAULT_POOL_CONFIG = {
+    min: 0,
+    max: DEFAULT_POOL_MAX,
+    acquireTimeoutMillis: 15000,
+    idleTimeoutMillis: 2000,
+};
+
+function wirePoolDebug(knex, name) {
```

ESLint

[error] Missing space before function parentheses.

(@typescript-eslint/space-before-function-paren)